.github/workflows/bump-version.yaml

```diff
@@ -43,6 +43,7 @@ jobs:
         if: steps.tag_version.outputs.new_version
         run: |
           git checkout -b bot/v${{ steps.tag_version.outputs.new_version }}
+          git push -d origin bot/v${{ steps.tag_version.outputs.new_version }}
```

Comment:
Will this fail if the branch doesn't exist?

Reply:
It will not. It's similar to the translations workflow https://github.com/openedx/tutor-contrib-aspects/blob/main/.github/workflows/compile-translations.yaml#L36